scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/clazz/MethodDefinition.scala

```diff
@@ -30,7 +31,13 @@ sealed trait MethodDefinition {
 
   protected def isValidMethodInfo(arguments: List[TypingResult], methodTypeInfo: MethodTypeInfo): Boolean = {
     val checkNoVarArgs = arguments.length >= methodTypeInfo.noVarArgs.length &&
-      arguments.zip(methodTypeInfo.noVarArgs).forall { case (x, Parameter(_, y)) => x.canBeSubclassOf(y) }
+      arguments.zip(methodTypeInfo.noVarArgs).forall {
+        // Allow pass array as List argument because of array to list auto conversion:
+        // pl.touk.nussknacker.engine.spel.internal.ArrayToListConverter
+        case (tc @ TypedClass(klass, _), Parameter(_, y)) if klass.isArray =>
+          Typed.genericTypeClass[java.util.List[_]](tc.params).canBeSubclassOf(y)
```

Comment:
I would add `tc.canBeSubclassOf(y) || ...` just in case if method takes array

Reply:
Done